lib/qms_web/controllers/auth_controller.ex

```diff
@@ -0,0 +1,44 @@
+defmodule QmsWeb.AuthController do
+  use QmsWeb, :controller
+
+  alias Qms.Repo
+  alias Qms.User
+  alias Qms.Spotify
+
+  def create(conn, params) do
```

Comment:
@mattpatterson94 Did you think about using pattern matching for the params which would allow you to split out the valid and invalid cases to their own functions?